.github/workflows/build-images-ci.yaml

```diff
@@ -147,7 +147,7 @@ jobs:
             quay.io/${{ github.repository_owner }}/${{ matrix.name }}-ci:latest-race
             quay.io/${{ github.repository_owner }}/${{ matrix.name }}-ci:${{ steps.tag.outputs.tag }}-race
           build-args: |
-            BASE_IMAGE=quay.io/cilium/cilium-runtime:eaa047d20a1736c81bcd4cab603f6c47c5508b21@sha256:ec480391acffe3c30a05b6c3b583f28caa9d20f25f33a650416454b29dabf642
+            BASE_IMAGE=quay.io/cilium/cilium-runtime:0da282e47af1907e17a1aa8dba3d41de6879ef62
```

Comment:
That seems incorrect:
```
$ docker pull quay.io/cilium/cilium-runtime:0da282e47af1907e17a1aa8dba3d41de6879ef62
Error response from daemon: manifest for quay.io/cilium/cilium-runtime:0da282e47af1907e17a1aa8dba3d41de6879ef62 not found: manifest unknown: manifest unknown
```

Reply:
I think I was too eager when updating the PR. I've removed the image digest updates and force-pushed just the changes, so that the image can be built by the pipeline. It looks like I have to run through the full digest update process again, as the images changed in the mean time. Apologies for the chaos!

Reply:
Looks like everything is working again. :)